api/test/wfl/integration/terra_test.clj

```diff
@@ -0,0 +1,34 @@
+(ns wfl.integration.terra-test
+  (:require [clojure.test          :refer [deftest is testing]]
+            [wfl.service.terra :as terra]
+            [wfl.tools.fixtures :refer [with-temporary-gcs-folder]]))
+
+(def terra-url
+  "https://firecloud-orchestration.dsde-dev.broadinstitute.org")
+
+(def workspace-namespace
+  "general-dev-billing-account")
+
+(def workspace-name
+  "wfl-integration")
+
+(def method-configuration-name
+  "ExternalWholeGenomeReprocessing")
+
+(def method-configuration-namespace
+  "general-dev-billing-account")
+
+(def entity-type
+  "sample")
+
+(def entity-name
+  "NA12878_PLUMBING")
+
+(deftest test-terra-submission
+  (testing "A workflow is created for the entity"
+    (let [submission-id (terra/create-submission terra-url workspace-namespace workspace-name
```

Comment:
Thanks for adding the test. Makes it easier to see how it's meant to be used.

What's the difference between the namespace and the name? Does it make sense to combine these into a fully-qualified name? (same for method-configuration)

Reply:
The `workspace-namespace` corresponds to a gcloud billing project and can contain multiple workspaces. Here's one I set up in dev using the "general-dev-billing-account" namespace: https://bvdp-saturn-dev.appspot.com/#workspaces. I like the idea of combining the two parameters into one since they're just used in the submission url.

Reply:
Also it seems like the `method-configuration-namespace` will always be the same as the `workspace-namespace`. But they can be specified individually through the Terra API so I'm assuming there's a case where they can be different.